Api/Model/Action/Export.php

```diff
@@ -294,7 +294,7 @@ private function writeOrderXml(Order $order): self
         $this->_xmlData .= "\t<Items>\n";
         $this->_orderItem($order); //call to the order items function
         //Get the order discounts
-        if ($this->_importDiscount && $order->getDiscountAmount() != '0.0000') {
+        if ($this->_importDiscount && null !== $order->getDiscountAmount() && $order->getDiscountAmount() != '0.0000') {
```

Comment:
Is the syntax of `&& null !== ` correct?

Reply:
Yes it is valid but you can invert them if you want. It would give:
```
if ($this->_importDiscount && $order->getDiscountAmount() !== null && $order->getDiscountAmount() != '0.0000') {
```

Reply:
I think it's a little more readable inverted, so let's go with that.